website/docs/reference/datasources/querying-oracle.md

```diff
@@ -17,203 +21,88 @@ This page describes how to connect your application to your Oracle databases and
 If you want to connect to a locally hosted database, you can use a service like ngrok to expose it. For more information, see [How to connect to local database on Appsmith](/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith).
 :::
 
-To connect to your database, Appsmith needs the following parameters:
+<dl>
+  <dt><b>Host Address</b></dt>
+  <dd>The network location of your Oracle database. This can be a domain name or an IP address. To connect to a local Oracle database, see <a href="/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith"><b>Connect Local Database</b></a> for directions. </dd><br />
 
-* **Host Address:** Provide the hostname or IP address of your database server. If you are connecting to a self-hosted database, see [this guide](/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith).
+  <dt><b>Port</b></dt>
+  <dd>The port number to connect to on the server. </dd><br />
 
-* **Port:**  Provide the port on which to connect to your database.
+  <dt><b>Service Name</b></dt>
+  <dd>The service name for your database instance. </dd><br />
 
-* **Service Name:** Provide the service name for your database instance.
+  <dt><b>Authentication</b></dt>
+  <dd>The <b>Username</b> and <b>Password</b> for the user with which you are connecting to the database.</dd><br />
 
-* **Authentication:** Provide the **Username** and **Password** for the user with which you are connecting to the database.
-
-* **SSL Mode:** choose one of the following modes to set whether your queries use SSL to connect:
-  * **TLS:** Appsmith attempts to connect using TLS.
-  * **Disabled:** Appsmith won't attempt an SSL connection.
+  <dt><b>SSL Mode</b></dt>
+  <dd>Determines whether your queries use an SSL connection to communicate with the database.</dd><br />
+  <dd><i>Options:</i>
+    <ul>
+      <li><b>TLS:</b> Appsmith attempts to connect using TLS.</li>
+      <li><b>Disabled:</b> Appsmith won't attempt any sort of SSL connection.</li>
+    </ul>
+  </dd>
+</dl>
 
 :::caution important
 If you are a self-hosted user, you may need to whitelist the IP address of the Appsmith deployment on your database instance or VPC before connecting to a database.
```

Comment:
```suggestion
If you are a self-hosted user, you must whitelist the IP addresses 18.223.74.85 and 3.131.104.27 of the Appsmith deployment on your database instance before connecting to a database. See [Managing Internet Protocol Allowlist and Blocklist Rules](https://docs.oracle.com/en/cloud/get-started/subscriptions-cloud/mmocs/managing-internet-protocol-whitelist-and-blacklist-rules.html) for more details.```

Reply:
@rohan-arthur can you confirm if the link shared above would help users with whitelisting IP addresses? Thanks. cc: @spark-c